Dockerfile

```diff
@@ -1,3 +1,5 @@
+ARG UBUNTU_CODENAME=jammy
```

Comment:
Is this an ARG because it should be configureable? Should we then update FROM as well?

Reply:
i'm not sure, copied it over from the vox puppetserver build.
it is more to be a constant, and it is not (yet) planned to be configureable from the outside

Reply:
axch, there is also a `ubuntu:jammy` docker tag, yes sure, then we can use it in the `FROM`